doc/docs/master/concepts/pipeline-concepts/datum/glob-pattern.md

```diff
@@ -35,14 +27,15 @@ Let's list the glob patterns at your disposal. We will later illustrate their us
 | `/*/*`| Pachyderm processes **each filesystem object in each subdirectory as a separate datum**.|
 | `/**` | Pachyderm processes **each filesystem object in all directories and subdirectories as a separate datum**.|
 
-TODO: I would just say subset of the files / directories.
```

Comment:
This TODO wasn't addressed.

Reply:
I was referring to the section below, not above, with this TODO. The change you made above probably makes sense though. File / directory is more precise than filesystem object.